etc/ci/manifest_changed.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
-./mach test-wpt --manifest-update SKIP_TESTS > /dev/null
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if `./mach test-wpt --manifest-update SKIP_TESTS > /dev/null`
```

Comment:
I am not sure do we have to add "Try debug build..." and "Try release build..." at Line 6 and 8.
Does anyone have idea?